lisa/tools/stress_ng.py

```diff
@@ -12,6 +12,45 @@
 from .make import Make
 
 
+def get_stress_ng_config() -> Dict[str, int]:
+    """
+    Try to determine the desired stress_ng configuration from various sources.
+    This is evaluated at import time to set test requirements.
+    Returns dict with keys: node_count, cpu_count, memory_mb
+    """
+    config = {
+        "node_count": 4,
+        "cpu_count": 2,
+        "memory_mb": 2048,
+    }
+
+    # Parse LISA command line variables (-v variable:value)
+    import sys
```

Comment:
You can pass in variables to the test case, it doesn't need to rely on command line.